lib/character.rb

```diff
@@ -0,0 +1,8 @@
+
+class Character
+  attr_reader :id
+
+  def initialize(id)
+    @id = id
+  end
+end
```

Comment:
Created a class so that an index could be created without worrying about repeating characters